<launch>
	<arg name="host" default="localhost" />
	<arg name="world_frame_id" default="world_ned" />
	<arg name="agent_frame_id" default="car" />
	<node name="airsim_node" pkg="airsim_ros_pkgs" type="airsim_node" output="screen">
		<!-- ROS timer rates. Note that timer callback will be processed at maximum possible rate, upperbounded by the following ROS params -->
		<param name="is_vulkan" type="bool" value="false" /> 
		<param name="auto_control" type="bool" value="true" /> 
		<param name="update_airsim_img_response_every_n_sec" type="double" value="0.05" /> 
		<param name="update_airsim_control_every_n_sec" type="double" value="0.001" />
		<param name="update_lidar_every_n_sec" type="double" value="0.05" />
		<param name="host_ip" type="string" value="$(arg host)" />
		<param name="world_frame_id" type="string" value="$(arg world_frame_id)" />
	</node>

	<node name="controller_node" pkg="airsim_ros_pkgs" type="controller_node" output="screen">
	</node>

	<!-- Static transforms -->
	<include file="$(find airsim_ros_pkgs)/launch/static_transforms.launch"/>
	<!--node name="record_bag" pkg="rosbag" type="record" args="-o $(find airsim_ros_pkgs)/rosbag/ -a" output="screen"/ -->
</launch>
